src/config.json

```diff
@@ -45,6 +45,7 @@
     "satoshilabs.design"
   ],
   "blacklist": [
+    "QmUDBVyGwqKdSayk7kDKUaj9J41Ft1DWizcKUx5UmgMgGy",
```

Comment:
Does the live version of MetaMask correctly handle this?

Reply:
From testing the `check()` method, it does ignore the entry -- I haven't tested the `MetaMask/core` usage but that uses `PhishingDetector.check` directly so it _should_ be fine